pandas/formats/format.py

```diff
@@ -2260,6 +2260,35 @@ def _format_strings(self):
         return fmt_values
 
 
+def format_percentiles(percentiles):
+    '''
```

Comment:
Do you mean to put the check here *instead of* of the one above (inside `describe()`? Or add another one?

By the way, initially I thought about a slightly different solution to make the function more universal, allowing also non-unique, unsorted input:
```python
def format_percentiles(percentiles, strictly_increasing=False)

    if strictly_increasing is True:
        # assume guarantees are satisfied (no checks)
        # run the existing code
    else:
        # run a modified code 
```
It would preserve the shape of `percentiles`, allow multi-dimensional ndarrays, and different input entries in `percentiles` would be still distinct in output.